### DISCUSSION TITLE
Denoising about mutiphase organization

### DISCUSSION THREAD

**Discussion by xiaoyuanmao** [mtex-toolbox/mtex#1260] [Ask Anything]
Whether the denoising command only applies to single phase grain structure?  Errors always occur in a mutiphase organization.

**Comment by kilir** [comment]
Hi,
actually it works on more than just one phase. You should provide some more information / code on what you do and what the error message says.
Cheers,
Rüdiger

**Comment by xiaoyuanmao** [comment]
Hello，
      I tried again but failed.What is the cause of this? I need your help!
Error message: ill usage "phaseList/check SinglePhase"
"Your variable contains the phases: Ni-superalloy, Nb C


However, you are executing a command that is only permitted for a single phase!


Please read the chapter "select EBSD data" for how to restrict EBSD data or grains to a single phase."
----------------------------------------------------------------
Best wishes
Xiao yuanmao













At 2021-10-22 14:31:55, "Rüdiger Kilian" ***@***.***> wrote:

Hi,
actually it works on more than just one phase. You should provide some more information / code on what you do and what the error message says.
Cheers,
Rüdiger

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by zmichels** [comment]
I think you will need to share exactly the code/syntax you are using for folks to help troubleshoot/solve the problem (perhaps you can post your script?). I have never had such an error or problem smoothing/denoising a multiphase dataset, so it is likely something specific to the code you are trying to use.

Zach
